biodatasets/ntcir_13_medweb/ntcir_13_medweb.py

```diff
@@ -0,0 +1,343 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+NTCIR-13 MedWeb (Medical Natural Language Processing for Web Document) task requires
+to perform a multi-label classification that labels for eight diseases/symptoms must
+be assigned to each tweet. Given pseudo-tweets, the output are Positive:p or Negative:n
+labels for eight diseases/symptoms. The achievements of this task can almost be
+directly applied to a fundamental engine for actual applications.
+
+This task provides pseudo-Twitter messages in a cross-language and multi-label corpus,
+covering three languages (Japanese, English, and Chinese), and annotated with eight
+labels such as influenza, diarrhea/stomachache, hay fever, cough/sore throat, headache,
+fever, runny nose, and cold.
```

Comment:
can we add a description of the actual files that need to be downloaded here? see https://github.com/bigscience-workshop/biomedical/blob/master/biodatasets/n2c2_2018_track2/n2c2_2018_track2.py#L18

the pattern we've been working towards for local datasets is, 
* create a directory
* download all files needed for the dataset into the directory above
* pass the above directory into `load_dataset` via the `data_dir` arg
* have the dataset loader handle all unzipping/untarring etc...

Reply:
I've addressed these in [b763eb0](https://github.com/bigscience-workshop/biomedical/pull/432/commits/b763eb0f069a37134090e7a2ad64e80533fe23a6)

Reply:
thanks, great detail.